File: packages/sdk-ts/src/seeds.ts
// Seed strings consumed by the on-chain program. Must match `program/programs/rein/src/constants.rs`
// byte-for-byte. Exported as Buffers (ready to feed into PublicKey.findProgramAddressSync).

export const VAULT_SEED: Buffer = Buffer.from('vault');
export const POLICY_SEED: Buffer = Buffer.from('policy');
export const BLOCKLIST_SEED: Buffer = Buffer.from('blocklist');
export const COUNTER_SEED: Buffer = Buffer.from('counter');
export const RECEIPT_SEED: Buffer = Buffer.from('receipt');
export const STEP_UP_SEED: Buffer = Buffer.from('stepup');
export const RECEIPT_V2_SEED: Buffer = Buffer.from('receipt-v2');
export const UMBRA_REF_SEED: Buffer = Buffer.from('umbraref');

File: packages/sdk-py/rein/seeds.py
"""Seed strings consumed by the on-chain program.

Must match ``program/programs/rein/src/constants.rs`` byte-for-byte. The SDK
exposes them as ``bytes`` so the consumer can feed them straight into
``Pubkey.find_program_address``.
"""

from __future__ import annotations

VAULT_SEED: bytes = b"vault"
POLICY_SEED: bytes = b"policy"
BLOCKLIST_SEED: bytes = b"blocklist"
COUNTER_SEED: bytes = b"counter"
RECEIPT_SEED: bytes = b"receipt"
STEP_UP_SEED: bytes = b"stepup"
